pom.xml

```diff
@@ -781,6 +810,7 @@
             </activation>
             <properties>
                 <dependency.check.skip>false</dependency.check.skip>
+                <resolver.maven.plugin.phase>none</resolver.maven.plugin.phase>
```

Comment:
Why does the phase get set to `none` in this profile?

Reply:
during a CI build we run the resolver plugin via CLI so that we can get the version it resolves to and write it out to the pom file. This plugin resolves the version but doesn't update the pom file. I think @xli1996 added a proper skip property recently so I will update it to use that. I will also add a comment here describing why we skip it.

Reply:
i updated this with a comment as to why we do this.